src/lib/components/Pagination/Pagination.js

```diff
@@ -38,7 +38,8 @@ export default class Pagination extends Component {
     currentSize,
     totalResults,
     onPageChange,
-    options
+    options,
+    ...restParams
```

Comment:
nit: `restParams` -> `otherParams`/`extraParams`? It reminds me of REST APIs :relaxed:

for the `nit`, in case you did not read the [article](https://stackoverflow.blog/2019/09/30/how-to-make-good-code-reviews-better/)...

Reply:
I've read it:P Thanks !